addons/l10n_hu_edi/views/report_invoice.xml

```diff
@@ -20,7 +20,8 @@
                         <li><t t-out="o.company_id.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/></li>
                         <li>
                             <t t-if="not o.fiscal_position_id.foreign_vat and o.company_id.partner_id.l10n_hu_group_vat">Group Tax ID</t>
-                            <t t-else="else" t-out="o.company_id.account_fiscal_country_id.vat_label or 'Tax ID'"/>:
+                            <t t-set="default_vat_label">Tax ID</t>
+                            <t t-else="else" t-out="o.company_id.account_fiscal_country_id.vat_label or default_vat_label"/>:
```

Comment:
Sorry, is `default_vat_label` not set here because it's in the `t-if` while here we are in `t-else` ?

Reply:
hello, `default_vat_label` should be set in this case